### DISCUSSION TITLE
Support on multiple structures, and absolute coordiantes mode

### DISCUSSION THREAD

**Discussion by TimothyGrass** [SuperLlama88888/holoprint#17] [General]
Basically, its adapting Madhatter's large structure mode, as to facilitate the construction of large architectures

**Comment by SuperLlama88888** [comment]
I'm currently working on putting multiple structures into the same pack.
Regarding absolute coordinates mode, I don't understand how it works just yet. Once I watch Madd Hatter's videos and understand it, I will add it to HoloPrint.

**Comment by SuperLlama88888** [comment]
Multiple structure support has been added, except IDK how to make a material list carousel for each structure, so the material list is combined currently.
